apps/bsd/src/components/Button.tsx

```diff
@@ -22,6 +18,7 @@ interface StyledButtonProps
 
 const buttonStyles = css<StyledButtonProps>`
   border: none;
+  display: inline-block;
```

Comment:
Button already has this by default but label doesn't so I added this for LabelButton's purpose so container elements adjust their size properly